### PR TITLE
 Change reference OS to Debian (Dockerfile and more)

### PR DESCRIPTION
**Pull request by fgalan**
Now that  CentOS8 has reached EOL we should move to another image to build ContextBroker. This PR is a proof-of-concept that shows how to do it in debian and alpine.

Can be pulled from dockerhub as:

* telefonicaiot/fiware-orion:debian
* telefonicaiot/fiware-orion:alpine

Pending

- [x] Some FIXME in Dockerfile.alpine (maybe we leave them at the end, Alpine version will be provided a example "as is")
- [x] Dockerfile for CI to Debian
- [x] Modify documentation (reference distribution, installl from sources, etc.)
- [x] CNR

### CODE REVIEW

File: doc/manuals/deprecated.md [hunk 1]
```diff
@@ -71,7 +71,6 @@ A list of deprecated features and the version in which they were deprecated foll
 Although you are encouraged to use always the newest Orion version, take into account the following
 information in the case you want to use old versions:
 
-* Old RPMs (since Orion 0.1.1) are available at the [FIWARE yum repository](http://repositories.lab.fiware.org/repo/rpm/6/x86_64).
```
Comment (mapedraza) [inline doc/manuals/deprecated.md:74]: Maybe this entry can point to previous RPM repo

**Review by mapedraza — APPROVED** [review]
LGTM

**Comment by fgalan** [comment]
@fisuda this PR modifies several .md files. It would be great if you could synchronize the Japanese translation, please. Thanks!

**Comment by fisuda** [comment]
I sent the PR #4083. Thanks.

**Pull request merged**